### PR TITLE
docs: add instructions for releases, and publishing

### PR DESCRIPTION
**Pull request by TarikGul**
Added docs for releasing and publishing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
 # @substrate/dev
 
 JS/TS development environment configuration and dependencies for @substrate projects.
+
+## Release && Publishing
+
+You must first be a member, and developer within the npm @substrate org before being able to publish this package to npm. You must also have 2FA enabled.
```
Comment (emostov) [inline README.md:7]:
```suggestion
NPM prerequisites: You must be a member of the `@substrate` NPM org and must belong to the `developers` team within the org. (Please make sure you have 2FA enabled.)
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+1. Create a branch 'name-v0-5-0'.
+
+2. Manually update the changelog with the commits leading up to the current release.
+
+3. Update the version inside of the package.json (NOTE: Very important for NPM).
```
Comment (emostov) [inline README.md:13]: We should look into how to do this auto with yarn 2. Maybe polkadot-js has some hints?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+
+4. Send a PR up and merge into main/master. 
+
+    * example commit: `git commit -m 'chore(release): 0.5.0'`
+
+5. Pull main/master down and follow the commands below.
+
+```
```
Comment (emostov) [inline README.md:21]:
```suggestion
```bash
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+$ yarn npm login
+$ yarn npm publish
+```
+
+6. Setup the git tags with the following commands
+
+```
```
Comment (emostov) [inline README.md:28]:
```suggestion
```bash
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+$ git tag v0.5.0
+$ git push origin --tags
```
Comment (emostov) [inline README.md:30]:
```suggestion
$ git push origin v0.5.0
```
Maybe? Just to make sure we only push the tag we intentionally made

File: README.md [hunk 6]
```diff
@@ -1,3 +1,33 @@
...
+```
+
+7. Setup the tag release in the github repo.
```
Comment (emostov) [inline README.md:33]:
```suggestion
7. Create a release from the new tag on GitHub.com. 
```

**Review by emostov — APPROVED** [review]
Thanks for docs. Have a few suggestions. We should look into automating the stuff that `yarn publish` used to do.

File: README.md (hunk 5 above)
Comment (TarikGul) [reply README.md:30]: Nice! TIL you can push up via the exact tag. I just checked the docs to make sure to. Pretty cool!

**Comment by TarikGul** [comment]
Nice thanks for the review! Funny enough I had made a similar issue earlier [#31](https://github.com/paritytech/substrate-js-dev/issues/31), will close it though as your's has a more concise description.

File: README.md (hunk 2 above)
Comment (TarikGul) [reply README.md:13]: Yea I agree, I will follow up on that issue you created and see how we can change the workflow for this local repo as well.

File: README.md [hunk 7]
```diff
@@ -1,3 +1,33 @@
...
 
 JS/TS development environment configuration and dependencies for @substrate projects.
+
+## Release && Publishing
+
+NPM prerequisites: You must be a member of the `@substrate` NPM org and must belong to the `developers` team within the org. (Please make sure you have 2FA enabled.)
+
+1. Create a branch 'name-v0-5-0'.
+
+2. Manually update the changelog with the commits leading up to the current release.
```
Comment (dvdplm) [inline README.md:11]: commits: Maybe PR-level granularity is sufficient yes?

File: README.md [hunk 8]
```diff
@@ -1,3 +1,33 @@
...
+
+3. Update the version inside of the package.json (NOTE: Very important for NPM).
+
+4. Send a PR up and merge into main/master.
```
Comment (dvdplm) [inline README.md:15]: "Make a PR and merge into main/master"?

**Pull request merged**